underscore.js

```diff
@@ -1340,8 +1347,16 @@
 
   // Shortcut function for checking if an object has a given property directly
   // on itself (in other words, not on a prototype).
-  _.has = function(obj, key) {
-    return obj != null && hasOwnProperty.call(obj, key);
+  _.has = function(object, path) {
+    if (!_.isArray(path)) path = [path];
```

Comment:
I've updated with a version that uses the old code for the `!_.isArray` case. Another option, would be to keep the old version as a private function and continue to use it internally. That will avoid the array check for all internal uses. This would be similar to the approach I took for `shallowProperty()/_.property`.